.github/workflows/winget-submission.yml

```diff
@@ -1,9 +1,14 @@
 name: Submit Nushell package to Windows Package Manager Community Repository
 
 on:
-  workflow_dispatch:
   release:
-    types: [published]
+    types: [released]
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: 'Specific tag name'
+        required: true 
+        type: string
```

Comment:
Ah that sounds convenient if we have to get our hands dirty!

Reply:
I don't think the current on in the repo works, though. You would be passing an empty `$github.release.tag_name` to WingetCreate 😅

Reply:
The current winget release definitely works by submitting a release to the winget-pkgs repo. No question about that. The part that's not been working is that the msi's created with cargo wix are failing. So, I don't think this PR will help with that.

Also, are we supposed to update the tag_name description with every release?

Reply:
> Also, are we supposed to update the tag_name description with every release?

No, you don't have to worry about that.